src/Controller.h

```diff
@@ -26,7 +26,7 @@ namespace VLCB
 //
 struct VlcbMessage
 {
-  uint8_t len; // Value 0-7 or -1 for messages handled in CanTransport
+  uint8_t len; // Value 0-7 or FF for messages handled in CanTransport
```

Comment:
Ouch, did I forget to update the comment? Thanks for fixing it.

Reply:
No worries.